init/initialize.sh

```diff
@@ -97,6 +97,15 @@ sed -e "s;RSYNCDIR_SS;$RSYNCDIR_SS;" \
     -e "s;RSYNCDIR_NPAD;$RSYNCDIR_NPAD;" \
     -e "s;RSYNCDIR_PTR;$RSYNCDIR_PTR;" \
     $SLICEHOME/conf/rsyncd.conf.in > /etc/rsyncd.conf
+
+# Allow CIRA rsync access to the paris-traceroute module at their own sites
+for site in "yyz01 yyc01 yul01"; do
+    if echo $(hostname) | grep -q $site; then
+        HOSTS_ALLOW=$(grep 'hosts allow' $SLICEHOME/conf/rsyncd.conf.in)
+        echo "    ${HOSTS_ALLOW}, 192.211.124.208/28" > /etc/rsyncd.conf
```

Comment:
To the first comment, yes! Thank you for catching that.  Fixed.  To the second comment, based on Jordan's commits in the _rsync-ip-filtering_ branch, I infer that this change was to only be applied to the [paris-traceroute] module, since he even went so far as to indent it (which I've preserved).